app/Upgrades/Paths/V240Upgrade.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Upgrades\Paths;
+
+use CachetHQ\Cachet\Upgrades\AbstractUpgrade;
+
+/**
+ * This is the v2.4.0 upgrade class.
+ *
+ * @author James Brooks <james@alt-three.com>
+ */
+class V240Upgrade extends AbstractUpgrade
+{
+    /**
+     * The version number of the upgrade.
+     *
+     * @var string
+     */
+    protected $version = 'v2.4.0';
+
+    /**
+     * Run the upgrade path.
+     *
+     * @return bool
+     */
+    public function upgrade()
+    {
+        echo 'This would upgrade us to '.$this->version."\n";
```

Comment:
What sort of processes would occur here? It seems like the way it's being done it's going to end up having to be a 'next only' upgrade system? I'm confused.